jsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
There's a, uh, fair number of red squiggles now, but most of them are things we should probably be tracking.

Dunno about other editors, TS only runs in mine w/ a `js/tsconfig` available